src/cli.ts

```diff
@@ -12,17 +12,24 @@ const cli = meow(`
 	Commands
 		validate 
 			Options
-				-c --context  Your AsyncAPI specification
+				-c --context  context-name saved in the store
 				-w --watch Enable watchMode (not implemented yet)
+		context
+			current  show the current set context
+			list     show the list of all stored context
+			remove  <context-name> remove a context from the store
+			use <context-name>  set any context as current
+			add <context-name> <filepath> add/update new context
 
 	Examples
-	  $ asyncapi validate --context=specification.yml
+	  $ asyncapi context add dummy ./specification.yml
+	  $ asyncapi validate --context=dummy
 `, {
 	flags: {
 		context: {
 			alias: 'c',
 			type: 'string',
-			isRequired: true
+			isRequired: false
```

Comment:
for now, because we do not automatically read current context, this must be set to true because it is always needed